README.md

```diff
@@ -51,15 +41,81 @@ SELECT * FROM userlog WHERE ts >= previousTime() AND  ts <= currentTime() ; //ti
+cassandra.connection.port=9042
+
+cassandra.source.route.topic1=keyspace1.table1
+```
+
+
+| name              | description                | default value  |
+|--------           |----------------------------|-----------------------|
+| | **Connect Properties (for both Source and Sink)** | |
+| name              |  Unique name for the connector. Attempting to register again with the same name will fail.   |                              |
+| connector.class   | The Java class for the connector,  `com.tuplejump.kafka.connect.cassandra.CassandraSource` or `com.tuplejump.kafka.connect.cassandra.CassandraSink` |  |
+| tasks.max         |  The maximum number of tasks that should be created for this connector. The connector may create fewer tasks if it cannot achieve this level of parallelism. | |
+| | **Cassandra Properties (for both Source and Sink)** | |
+| cassandra.connection.host | The host name or IP address to which the Cassandra native transport is bound. | localhost |
+| cassandra.connection.port | Port for native client protocol connections. | 9042 |
+| cassandra.connection.auth.username | Cassandra username | '' |
+| cassandra.connection.auth.password | Cassandra password | '' |
+| cassandra.connection.timeout.ms | Connection timeout duration (in ms) | 8000 |
+| cassandra.connection.read.timeout | Read timeout duration (in ms) | 120000 |
+| cassandra.connection.keep_alive (not yet implemented) | Period of time, in ms, to keep unused connections open. |  |
+| cassandra.connection.reconnect.delay.min | Minimum period of time (in ms) to wait before reconnecting to a dead node. | 1000 |
+| cassandra.connection.reconnect.delay.max | Maximum period of time (in ms) to wait before reconnecting to a dead node. | 60000 |
+| cassandra.connection.compression (broken) | One of the compressor strategies. Can be DeflateCompressor, SnappyCompressor or LZ4Compressor | NONE |
+| cassandra.connection.consistency | Consistency level. Values allowed can be seen [here](http://docs.datastax.com/en/drivers/java/2.1/com/datastax/driver/core/ConsistencyLevel.html) | QUORUM |
+| cassandra.task.parallelism | Can be one of the values: one-to-one (One topic-to-keyspace.table per task) or one-to-many (N-topic-to-keyspace.table per task) | one-to-one |
+| | **Source Connect Properties** | |
+| cassandra.source.route.\<topic_name\> | The Select Query to get the data. (Refer CassandraSource documentation for more details) | |_
+| cassandra.source.poll.interval | Frequency in ms to poll for new data in each table. | 60000 |
+| cassandra.source.split.size (not yet implemented?) | Size of Cassandra data to be read in a single Kafka task;  determines the number of partitions. | 64 |
+| cassandra.source.fetch.size | Number of CQL rows to fetch in a single round-trip to Cassandra. | 1000 |
+| cassandra.source.consistency (not yet implemented?) | Consistency level for reads, defaults to LOCAL_ONE;  higher consistency level will disable data-locality.| LOCAL_ONE |
```

Comment:
Yes, this is also not implemented yet but I want the user to be able to pass in specific config per table/query if they need it. `cassandra.connection.consistency` is just a default if not set explicitly (coming soon).